app/views/unsupportedBrowserPage.scala.html

```diff
@@ -0,0 +1,61 @@
+                                <p class="component-body-copy__para">This is crucial, when like many media organisations, the Guardian is operating in an incredibly challenging commercial environment, and the advertising that we used to rely on to fund our work continues to fall.</p>
+                            </div>
+                        </div>
+                        <div class="component-video">
+                            <iframe title="scottTrustExplained" src="https://www.youtube.com/embed/tgusJhI95TI?enablejsapi=1&amp;wmode=transparent&amp;rel=0" frameborder="0" width="560" height="315" allowfullscreen="true"></iframe>
+                        </div>
+                    </div>
+                    <div class="why-support__bottom-content">
+                        <div class="component-body-copy">
+                            <p class="component-body-copy__para">We haven't put up a paywall – we want to keep our journalism as open as we can.</p>
+                            <p class="component-body-copy__para">We don’t have a billionaire owner pulling our strings. Our owner, the Scott Trust, safeguards our editorial independence from commercial or political interference and reinvests revenue into our journalism, as opposed to into shareholders' pockets.</p>
+                            <p class="component-body-copy__para">Help to make the Guardian's journalism possible: by funding it, by reading it, by sharing it, and by participating in it so that together we can continue to tell the stories that matter, to inform the world and to make it a better place. </p>
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <footer class="component-footer">
+                <div class="component-footer__content gu-content-margin">
+                    <a class="component-footer__privacy-policy" href="https://www.theguardian.com/help/privacy-policy">Privacy Policy</a>
+                    <small class="component-footer__copyright">© 2017 Guardian News and Media Limited or its affiliated companies. All rights reserved.</small>
+                    <p class="component-contrib-legal">The ultimate owner of the Guardian is The Scott Trust Limited, whose role it is to secure the editorial and financial independence of the Guardian in perpetuity. Reader contributions support the Guardian’s journalism. Please note that your support of the Guardian’s journalism does not constitute a charitable donation, as such your contribution is not eligible for Gift Aid in the UK nor a tax-deduction elsewhere. If you have any questions about contributing to the Guardian, please <a href="mailto:contribution.support@@theguardian.com">contact us here</a>.</p>
```

Comment:
`@@theguardian.com`? Is that a twirl thing?

Reply:
It compiled-bombed with @guardian is not available 😆